app/backend/app.py

```diff
@@ -308,9 +308,10 @@ async def setup_clients():
         else:
             endpoint = f"https://{AZURE_OPENAI_SERVICE}.openai.azure.com"
 
-        # Store on app.config for later use inside requests
+        api_version = os.environ["AZURE_OPENAI_API_VERSION"]
```

Comment:
This will cause a key error in old environments, please use get() and provide a default value.

Reply:
The latest. I just need to make sure it works with gpt-4-vision but I'll test that before merging.